openmc/mgxs_library.py

```diff
@@ -1644,6 +1644,54 @@ def set_multiplicity_matrix_mgxs(self, nuscatter, scatter=None,
         self._multiplicity_matrix[i] = \
             np.nan_to_num(self._multiplicity_matrix[i])
 
+    def set_inverse_velocity_mgxs(self, inverse_velocity, temperature=294.,
+                                  nuclide='total', xs_type='macro',
+                                  subdomain=None):
+        """This method allows for an openmc.mgxs.InverseVelocity
+        to be used to set the absorption cross section for this XSdata object.
+
+        Parameters
+        ----------
+        inverse_velocity : openmc.mgxs.InverseVelocity
+            MGXS Object containing the inverse velocity for the domain of
+            interest.
+        temperature : float
+            Temperature (in Kelvin) of the data. Defaults to room temperature
+            (294K).
+        nuclide : str
+            Individual nuclide (or 'total' if obtaining material-wise data)
+            to gather data for.  Defaults to 'total'.
+        xs_type: {'macro', 'micro'}
+            Provide the macro or micro cross section in units of cm^-1 or
+            barns. Defaults to 'macro'.
+        subdomain : iterable of int
+            If the MGXS contains a mesh domain type, the subdomain parameter
+            specifies which mesh cell (i.e., [i, j, k] index) to use.
+
+        See also
+        --------
+        openmc.mgxs.Library.create_mg_library()
+        openmc.mgxs.Library.get_xsdata
```

Comment:
It should have parentheses. This is used for all the setter methods, so I'll change it throughout.